### ISSUE TITLE
Support Importing Data from Kafka

### ISSUE THREAD

**Issue by MrSanZhi** [openGemini/openGemini#38]
Kafka is an advanced system for a replica, shared-multiple, multiple-producer storage, multiple-subscriber. 
Kafak has the advantages of high throughput, high performance, persistent data storage, easy to scale out.
With the development of cloud technology, it is very popular in the Devops and IOT scenes. 
Therefore, developing an intermediate component to import data from kafka to openGemini is particularly convenient for users.

**Comment by xiangyu5632** [comment]
Agree. However, consider how to convert various data formats in Kafka to openGemini's line protocol.
I think you can design the measurement, tag, and field types in the connector, which are filled by developers.
